vendor/bat-native-ledger/include/bat/ledger/publisher_info.h

```diff
@@ -12,9 +12,16 @@
 #include <utility>
 
 #include "bat/ledger/export.h"
+#include "bat/ledger/public/interfaces/ledger.mojom.h"
 
 namespace ledger {
 
+using PublisherInfo = mojom::PublisherInfo;
+using PublisherInfoPtr = mojom::PublisherInfoPtr;
+using PublisherInfoList = std::vector<PublisherInfoPtr>;
+// TODO - remove this
+using PublisherInfoListStruct = PublisherInfoList;
```

Comment:
seems like a left over here?

Reply:
I'm not sure if I removed all references to it